src/Components/Header/index.jsx

```diff
@@ -0,0 +1,35 @@
+import "bootstrap-icons/font/bootstrap-icons.css"
+import { useState } from "react"
+
+
+const Header = ({inboxCounter}) => {
+
+    const [displayMenu, setDisplayMenu] = useState(false)
+
+    return (
+        <>
+            <div className="col-12 d-flex justify-content-between align-items-center border border-secondary bg-secondary text-white">
+                <div className="col-2 d-md-none">
+                    <button onClick={() => {setDisplayMenu(!displayMenu)}} className="mx-3 border border-black btn btn-secondary text-center" type="button" data-bs-toggle="dropdown" aria-expanded="false">MENU</button>
```

Comment:
you dont need the `data-bs-toggle` - thats for bootstrap Js which your not using. You should also set the `aria-expanded` value to be the `displayMenu` state